js/zb.js

```diff
@@ -2446,6 +2448,86 @@ module.exports = class zb extends Exchange {
         return result;
     }
 
+    async modifyMarginHelper (symbol, amount, type, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        amount = this.numberToString (amount);
+        const position = this.safeString (params, 'positionsId');
+        const request = {
+            'positionsId': position,
```

Comment:
If `positionsId` is required by the exchange, then we must throw ArgumentsRequired if it is not defined in params.